fvm/evm/evm_test.go

```diff
@@ -199,7 +198,9 @@ func TestBridgedAccountWithdraw(t *testing.T) {
                                    let bridgedAccount <- EVM.createBridgedAccount()
                                    bridgedAccount.deposit(from: <-vault)
 
-                                   let vault2 <- bridgedAccount.withdraw(balance: EVM.Balance(flow: 1.23))
+								   let bal = EVM.Balance(0)
+								   bal.setFLOW(flow: 1.23)
+                                   let vault2 <- bridgedAccount.withdraw(balance: bal)
```

Comment:
minor: some mixing of tabs and spaces.

Reply:
sorry I have to https://www.youtube.com/watch?v=SsoOG6ZeyUI 
:D

Reply:
fixed